apps/sms/test/marionette/lib/views/shared/composer_accessors.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/* global module */
+
+var SELECTORS = Object.freeze({
+  messageInput: '#messages-input',
+  subjectInput: '.subject-composer-input',
+  sendButton: '#messages-send-button',
+  attachButton: '#messages-attach-button',
+  charCounter: '.message-counter',
+  mmsLabel: '.mms-label',
+  attachment: '#messages-input .attachment-container'
+});
+
+function ComposerAccessor(client) {
+  this.client = client;
+}
+
+ComposerAccessor.prototype = {
+  get messageInput() {
+    return this.client.helper.waitForElement(SELECTORS.messageInput);
+  },
+
+  get subjectInput() {
+    return this.client.findElement(SELECTORS.subjectInput);
+  },
+
+  get sendButton() {
+    return this.client.helper.waitForElement(SELECTORS.sendButton);
```

Comment:
```waitForElement``` waits for the element to become _visible/displayed_ [1]. I usually _try_ to use ```findElement``` for the elements that can be hidden/removed during ordinary app flow, and ```waitForElement``` for the elements that should be definitely visible in the DOM tree.

[1] https://github.com/mozilla-b2g/gaia/blob/e42f7e9937b62f84d2baef547f5bd08e1a6ac44b/tests/jsmarionette/plugins/marionette-helper/index.js#L144